leverage/container.py

```diff
@@ -379,10 +385,16 @@ class AWSCLIContainer(LeverageContainer):
 
     AWS_CLI_BINARY = "/usr/local/bin/aws"
 
+    # SSO scripts
     AWS_SSO_LOGIN_SCRIPT = "/root/scripts/aws-sso/aws-sso-login.sh"
     AWS_SSO_LOGOUT_SCRIPT = "/root/scripts/aws-sso/aws-sso-logout.sh"
     AWS_SSO_CONFIGURE_SCRIPT = "/root/scripts/aws-sso/aws-sso-configure.sh"
 
+    # SSO constants
+    AWS_SSO_LOGIN_URL = "https://device.sso.{region}.amazonaws.com/?user_code={user_code}"
+    AWS_SSO_CODE_WAIT_SECONDS = 1
```

Comment:
```suggestion
    AWS_SSO_CODE_WAIT_SECONDS = 2
```
I'd go with 2 here as I think the feedback message is enough to calm the user's anxiety after running the 'configure sso' command.